benchmark/path/normalize-posix.js

```diff
@@ -6,12 +6,13 @@ const bench = common.createBenchmark(main, {
   path: [
     '',
     '.',
+    '/',
     '/../',
     '/foo',
     '/foo/bar',
     '/foo/bar//baz/asdf/quux/..',
   ],
-  n: [1e5]
```

Comment:
I am against raising this limit or to add another edge case. The same applies to the other benchmark. These benchmarks are meant to run in a short time.